ios/gamma/split-view/RNSSplitViewScreenController.swift

```diff
@@ -90,10 +146,19 @@ public class RNSSplitViewScreenController: UIViewController {
         "[RNScreens] Expected to find RNSSplitViewHost component for RNSSplitViewScreen component"
       )
 
-      shadowStateProxy.updateShadowState(
-        ofComponent: splitViewScreenComponentView,
-        inContextOfAncestorView: ancestorView!
-      )
+      if let currentAnimationFrame = animationFrame {
+        let localOrigin = splitViewScreenComponentView.convert(
+          splitViewScreenComponentView.frame.origin, to: ancestorView)
+        let convertedFrame = CGRect(origin: localOrigin, size: currentAnimationFrame.size)
+
+        shadowStateProxy.updateShadowState(withFrame: convertedFrame)
+        animationFrame = nil
```

Comment:
Why are you setting it to `nil` here?

Reply:
Note: I needed to revisit the exit criteria for stopping DisplayLink so this was fixed as a part of:

https://github.com/software-mansion/react-native-screens/pull/3073/commits/bdbd501fd2ab312203f96d6dba41ef5afe0f4f7f